libvast/vast/view.hpp

```diff
@@ -244,24 +264,32 @@ class container_view_iterator
   }
 
 private:
-  View view_;
+  container_view_ptr<T> view_;
```

Comment:
Iterators usually do not participate in the lifetime of an object. What is the rationale behind having owning iterators?

Reply:
Good call, that was merely an oversight.